notebook/templates/notebook.html

```diff
@@ -264,7 +267,7 @@
 
                         {% for helplinks in sections %}
                             {% for link in helplinks %}
-                                <li><a href="{{link[0]}}" {{'target="_blank" title="Opens in a new window"' if link[2]}}>
+                                <li><a {% block helplink_attributes %}{% endblock %} href="{{link[0]}}" {{'target="_blank" title="Opens in a new window"' if link[2]}}>
```

Comment:
like below, perhaps `class=help_link` so that it can be treated with CSS rather than a block.